src/main/kotlin/com/sourcegraph/cody/config/notification/AccountSettingChangeListener.kt

```diff
@@ -60,14 +55,9 @@ class AccountSettingChangeListener(project: Project) : ChangeListener(project) {
 
             // Log install events
             if (context.serverUrlChanged) {
-              GraphQlLogger.logInstallEvent(project).thenAccept { e ->
-                codyApplicationSettings.isInstallEventLogged = e
-              }
-            } else if (context.accessTokenChanged &&
-                !codyApplicationSettings.isInstallEventLogged) {
-              GraphQlLogger.logInstallEvent(project).thenAccept { e ->
-                codyApplicationSettings.isInstallEventLogged = e
-              }
+              GraphQlLogger.logCodyEvent(project, "settings.serverURL", "changed")
+            } else if (context.accessTokenChanged) {
+              GraphQlLogger.logCodyEvent(project, "settings.accessToken", "changed")
```

Comment:
I couldnt find these on a quick check being emitted for vscode, is this an expected deviation?

Reply:
See some discussion here. I think this is okay! https://github.com/sourcegraph/jetbrains/issues/43#issuecomment-1887944522